src/test/librbd/test_librbd.cc

```diff
@@ -4232,6 +4232,68 @@ interval_set<uint64_t> round_diff_interval(const interval_set<uint64_t>& diff,
   return rounded_diff;
 }
 
+TEST_F(TestLibRBD, TestSizeImageSnap)
+{
+  rados_ioctx_t ioctx;
+  rados_ioctx_create(_cluster, m_pool_name.c_str(), &ioctx);
+
+  rbd_image_t image;
+  int order = 0;
+  std::string image_name = get_temp_image_name();
+  uint64_t size = 16 << 20;
+  uint64_t features;
+  features = (RBD_FEATURE_EXCLUSIVE_LOCK |
+              RBD_FEATURE_OBJECT_MAP |
+              RBD_FEATURE_FAST_DIFF);
+  ASSERT_EQ(0, create_image_full(ioctx, image_name.c_str(), size, &order,
+				 false, features));
```

Comment:
Go back to what you had before here (i.e. just `create_image`) and instead add `REQUIRE_FEATURE(RBD_FEATURE_FAST_DIFF);` at the top of the test.

Reply:
I considered that and doing that skips the test.

Reply:
Done